pvlib/iotools/sodapro.py

```diff
@@ -363,7 +360,7 @@ def read_cams(filename, integrated=False, label=None, map_variables=True):
 
     See Also
     --------
-    pvlib.iotools.parse_cams, pvlib.iotools.get_cams
+    pvlib.iotools.parse_cams,pvlib.iotools.get_cams
```

Comment:
You can go ahead and put this space back since this was not the trailing white space.